### PR TITLE
fix: Update description & image for metadata

### LINKED ISSUE #137: Docs nicer discord embed
## What needs to be done
When sharing the docs site on discord, there is an automatic discord embed that is generated. For example, if someone were to share our github foundation page on discord, they get an embed that looks like this:

image

Our current auto generated embed looks like this and could use some work (maybe better messaging, more descriptions, logo, use of our theme colors)

image

## Support
In case you need any kind of help (such as getting your environment setup or asking questions about our tech) don’t be afraid to ask in our [discord](https://discord.atsign.com). There is a team full of developers ready to jump in and help on whatever your issue may be.

### PR DESCRIPTION
**Pull request by daniloff200**
closes #137

**- What I did**
Updated the description, added the image

**- How to verify it**
We need to deploy that and check on discord
 
Some info for changelog:
Update the description, add the image in metadata

### CODE REVIEW

File: content/docs/_index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ lead: "Learn all about how the atPlatform works" # The lead below the title (ON
 
 description:
   | # SEO Description of the page (Shows in google and atsign.dev search)
-  Documentation for the atPlatform.
+  Documentation for the atPlatform of atSign Foundation. Learn how to use our tools and SDK!
```
Comment (xavierchanth) [inline content/docs/_index.md]: Please limit your PR to one set of changes.

File: config/_default/config.yaml [hunk 2]
```diff
@@ -23,8 +27,7 @@ taxonomies:
 
 # Relative Permalinks
 # permalinks:
-#   blog: /blog/:title/
-
+#   blog: /blog/:title/Т
```
Comment (xavierchanth) [inline config/_default/config.yaml]: Please revert this change.

File: config/_default/config.yaml [hunk 3]
```diff
@@ -10,6 +10,10 @@ languageCode: en
 paginate: 7
 rssLimit: 10
 
+params:
```
Comment (xavierchanth) [inline config/_default/config.yaml]: Please move to `/config/_default/params.yaml`

**Review by xavierchanth — CHANGES_REQUESTED** [review]

File: content/docs/_index.md (hunk 1 above)
Comment (daniloff200) [reply content/docs/_index.md]: Well, the initial task was also to update the description in some... more prettier way, so, I decided to change that too

File: config/_default/config.yaml (hunk 3 above)
Comment (daniloff200) [reply config/_default/config.yaml]: done

File: content/docs/_index.md (hunk 1 above)
Comment (JeremyTubongbanua) [reply content/docs/_index.md]: Yes @XavierChanth I asked for a better description in the discord embed in the issue card #137 

Does this change reflect the discord embed @daniloff200 ?

Comment (xavierchanth) [reply content/docs/_index.md]: Discord uses [opengraph](https://ogp.me/) for their embeds, [see here](https://gohugo.io/templates/internal/#configure-open-graph) for how to implement the metadata in Hugo.

Comment (daniloff200) [reply content/docs/_index.md]: hi, @XavierChanth . Yeah, I've also seen that doc. I should say, it's not completely working for this case, because, each .md file has its own description & title, which overrides the og tags

Comment (daniloff200) [reply content/docs/_index.md]: I should remove that, I think, because, we need to change bunch of titles, if we want to rework them, not only this one

Comment (xavierchanth) [reply content/docs/_index.md]: > hi, @XavierChanth . Yeah, I've also seen that doc. I should say, it's not completely working for this case, because, each .md file has its own description & title, which overrides the og tags

Are you sure that it overrides the og tags? As far as I know these are different from the basic meta tags (which are modified on a per-page basis).

Comment (daniloff200) [reply content/docs/_index.md]: I can show u :) 
On my video you can see, that some changes in config was not set, but, changes in title / description take place not only on the page, but, also for OG tags
[Untitled_ Oct 5 2022 10_24 PM.webm](https://user-images.githubusercontent.com/13692220/194145233-be90bae9-fa59-4d4f-a706-3f083ffea4fe.webm)

Comment (xavierchanth) [reply content/docs/_index.md]: Thanks for the insight, that was not the behavior I expected, but after thinking about it, it makes sense. I will resolve and approve this, and open a new issue to update the descriptions across the various pages.

**Review by xavierchanth — APPROVED** [review]

File: content/docs/_index.md (hunk 1 above)
Comment (daniloff200) [reply content/docs/_index.md]: Thanks! Yeah, that was also kinda unexpectable for me, but, it is what it is in our work. Every day you learn something new, sometimes some strange stuff ;)

**Comment by daniloff200** [comment]
@XavierChanth  hey :)
Can this one be merged, or, some additional changes are required here?

**Comment by JeremyTubongbanua** [comment]
@daniloff200 
Hey! Thank you for your contribution. Your PR was accepted, merged, and it closed an issue. You could be eligible for a prize.

To be eligible, please send an email to [hacktoberfest@atsign.com](mailto:hacktoberfest@atsign.com) with a screenshot of the PR approval email. This is to prove that your email corresponds to the correct GitHub account. Thank you!

When it comes time to distribute the prizes, we will look at all the emails we've received from that email so don't forget to do this!
